django_mailbox/models.py

```diff
@@ -372,6 +382,11 @@ def get_new_mail(self, condition=None):
         for message in connection.get_message(condition):
             msg = self.process_incoming_message(message)
             new_mail.append(msg)
+        self.last_polling = now()
+        if django.VERSION >= (1, 5):  # Django 1.5 introduces update_fields
+            self.save(update_fields=['last_polling'])
```

Comment:
I think somebody can modify instance any way before execute get_new_mail. I would like reduce side-effects of execution function. I think main effect of function is get mail, but there's a side-effect which IMO should be as small as possible.